### ISSUE TITLE
The lantasy splash page isn't displaying fully in mobile

### ISSUE THREAD

**Issue by wallacetech** [BCGamer/website#9]
The date is cut off on my iphone.

**Comment by cdobbyn** [comment]
Fixed in https://github.com/BCGamer/website/commit/a486662

**Comment by cdobbyn** [comment]
This issue is a regression after implementing CMS page. Assigned to Adrian to fix.

**Comment by cdobbyn** [comment]
This was originally fixed by changing the title to "em" unit of measure instead of "px". See previous commits on Master for an example.

**Comment by cdobbyn** [comment]
I think this is it. It makes it so the text is re-sized when the media size is smaller than 440px.
```
@media only screen and (max-width: 440px) {
    .event-info .sl{
        font-size: 16vw;
    }
    .event-info .fl{
        font-size: 8vw;
        margin-bottom:0px;
    }
    .event-info .tl{
        font-size: 4vw;
        margin-top: 0px;
        margin-bottom:0px;
    }
    .event-info .event-date{
        font-size: 4vw;
        margin-top: 0px;
    }
}
```

**Comment by cdobbyn** [comment]
Fix in commit 4df661f
